_1327/documents/urls.py

```diff
@@ -19,6 +19,7 @@
 	url(r"(?P<title>[\w-]+)/edit$", views.edit, name='edit'),
 	url(r"(?P<title>[\w-]+)/autosave$", views.autosave, name='autosave'),
 	url(r"(?P<title>[\w-]+)/publish$", views.publish, name='publish'),
+	url(r"(?P<title>[\w-]+)/publish_student$", views.publish_student, name='publish_student'),
```

Comment:
wouldn't it be better to do this using only one url endpoint and distinguishing between students and university network using a POST/GET param?